api/routes/apds/submitted/get.js

```diff
@@ -0,0 +1,32 @@
+const logger = require('../../../logger')('submitted apds route get');
+const defaultApdModel = require('../../../db').models.apd;
```

Comment:
I did not know you could do dot notation on `require`d elements! whoooa

Reply:
eslint will complain about it if you're requiring into a variable with the same name as the last dotted thing.  It'll want you to use the bracket thingy instead:

```js
```